jax/_src/lax/parallel.py

```diff
@@ -1487,7 +1487,7 @@ def _pgather_impl(src, idx, *, axes):
   non_axes_shape = src_axes_front.shape[len(axes):]
   src_one_axis_front = src_axes_front.reshape((-1,) + non_axes_shape)
   slice_sizes = (1,) + non_axes_shape
-  idx = lax.reshape(idx, idx.shape + (1,))
+  idx = lax_numpy.expand_dims(idx, -1)
```

Comment:
Let's use `lax.expand_dims` here instead, since this is a submodule of `lax`.

Reply:
Cool, I didn't know that `lax.expand_dims` also supports negative `axis` arguments. I changed it and updated the PR.